docs/en_US/edgex/edgex_rule_engine_tutorial.md

```diff
@@ -77,6 +77,44 @@ c78419bc5096   consul:1.9.5                                                    "
 d4b236a7b561   redis:6.2.4-alpine                                              "docker-entrypoint.s…"   13 minutes ago   Up 13 minutes   127.0.0.1:6379->6379/tcp                                                               edgex-redis
 ```
 
+#### Connection reuse
+when eKuiper get data from messageBus and send back the processed result, user needs to specify the connection info separately when create the source and sink.
```

Comment:
Add a new line and use bigger case 'When'.  'get' should be 'gets'.  'When create' -> 'When creating'